### DISCUSSION TITLE
[Question] New resources/view

### DISCUSSION THREAD

**Discussion by Hugo1380** [simonschaufi/laravel-dkim#5] [General]
Hi,

My question is the following, what is the new views that is created when we publish the config file ?

Apparently related to `/vendor/laravel/framework/src/Illuminate/Mail/resources/views`. Is it necessary for your service provider or I can remove it ?

**Comment by simonschaufi** [comment]
Hi, this package is only about sending emails, I don't care about the email templates. There is no config about "new views". Does this answer your question?

**Reply by Hugo1380** [reply]
So the templates that is generate by your service provider can be delete ?

<img width="239" alt="Capture d’écran 2021-06-16 à 15 20 18" src="https://user-images.githubusercontent.com/9752754/122227517-3cd71e00-ceb7-11eb-98e1-4a2ccd5043fc.png">

This files was generated after I publish the config.

**Reply by simonschaufi** [reply]
The templates probably get exported because I just extend the MailServiceProvider from the framework which does the same but they don't come from MY service provider. You can just delete them again if you don't need them.

**Reply by Hugo1380** [reply]
Thank you, that what I wanted to know, if your service provider needed that to work correctly. I have my answer thanks :)

**Reply by simonschaufi** [reply]
Does this package help you in your business? If you want to appreciate my work that I put into this package, I would appreciate a small donation! Links are on my website which is linked in the project. Thank you for using it!
